content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -124,38 +132,57 @@ based on formulas.
 Check out the :ref:`Advanced price rules (discounts, formulas) <sales/advanced-price-rules>` section
 for detailed steps on how to add advanced price rules to a pricelist.
 
-Time-based rules tab
+Recurring Prices tab
 --------------------
 
-Time-based rules are used specifically with :doc:`subscription products
-</applications/sales/subscriptions/products>`. Be sure to check out the Odoo *Subscriptions*
-:doc:`documentation </applications/sales/subscriptions>`.
+Recurring prices are specifically used with :doc:`subscription products
+</applications/sales/subscriptions/products>`.
+
+.. seealso::
+   :doc:`/applications/sales/subscriptions`
 
-Under the :guilabel:`Time-based rules` tab, the same functionality of the :guilabel:`Price Rules`
-tab is present, with the only difference being that a repeating time period can be applied in the
-:guilabel:`Period` column.
+Under the :guilabel:`Recurring Prices` tab, the same functionality of the :guilabel:`Price Rules`
+tab is present. The only difference being that a recurring time period can be applied in the
+:guilabel:`Recurring Plan` column.
 
-After a :guilabel:`Product` and potential :guilabel:`Variant` are selected in the
-:guilabel:`Time-based rules` tab, select the blank field in the :guilabel:`Period` column to reveal
-a drop-down menu of pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).
+Once :guilabel:`Products` and/or :guilabel:`Product Variants` are selected, click :guilabel:`Add a
+price rule`, and select the blank field in the :guilabel:`Recurring Plan` column to reveal a
+drop-down menu of pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).
 
-New recurrence periods can also be directly created from this column, by typing in the new name for
-the :guilabel:`Period` and then selecting :guilabel:`Create` to create the time period, which can be
-edited later. Or, select :guilabel:`Create and edit...` to reveal a pop-up form, in which the new
-recurrence period can be directly configured.
+New recurrence periods can also be created from this column. To do so, type in the name for the new
+:guilabel:`Recurring Plan`, then select :guilabel:`Create` from the resulting drop-down menu to
+create the time period, which can be edited later.
+
+Or, select :guilabel:`Create and edit...` to reveal a :guilabel:`Create Recurring Plan` pop-up form.
 
 .. image:: pricing/time-period-popup.png
    :align: center
    :alt: Custom time period pop-up form in Odoo Sales.
 
-From this :guilabel:`Create Period` pop-up form, add a :guilabel:`Name`, :guilabel:`Duration`, then
-a :guilabel:`Unit` (e.g. `Days`, `Weeks`, etc.). When finished, click :guilabel:`Save and Close`.
+From this pop-up form, the new recurrence period can be configured, with specific
```

Comment:
would move this to the same paragraph as line 156 since it's a part of that supplemental instruction, otherwise it comes off as a new instruction in the main flow (which it's not, it's an aside)